docs/developing.rst

```diff
@@ -571,6 +571,19 @@ Example::
 
             toil.exportFile(outputFileID, 'file:///some/other/local/path')
 
+
+There are some trade-offs to file staging. For example it may be more efficient to directly download
+a large file if it is used once. Importing will be more efficient if the file is needed several
+times during a workflow and the job store's storage platform supports certain optimizations. For
+example in S3 importing should be used if the source file is in a different region than the job
+store. Note that toil-scripts/lib contains common functionality for downloading URLs (see
+https://github.com/BD2KGenomics/toil-scripts/blob/master/src/toil_scripts/lib/urls.py) in cases
+where a file is used once. Also, per-sample files should be imported on a worker not the leader.
+This can be done inside job functions with :func:`job.FileStore.importFile`. Using this method will
```

Comment:
I don't see ```meth``` anywhere in this file. Other methods also use ```fund``` like 
```
e.g. :func:`toil.job.Job.addChild` 
and :func:`toil.job.Job.addFollowOn`. 
```
But I can change it to ```meth``` if those are technically incorrect